django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def bind_to_form(self, form, name):
```

Comment:
I was thinking:
```python
def bind_to_form(self, form, name, **kwargs):
    return BoundField(form, self, name, **kwargs)
```
but I'm not convinced this is a good practice for a "just in case" scenario, so we can probably drop the idea.